ui/model/game_history.py

```diff
@@ -100,10 +100,12 @@ def get_player_history_string(self, player: Color):
             history = player_2_history
         for i in range(0, len(history)):
             history_string += \
-                str(i+1) + "." + "\n" + \
-                self.get_player_total_time(i-1, player) + " >>> " + \
-                self.get_player_total_time(i, player) + "\n" + \
-                str(history[i].move) + "\n"
+                str(len(history) - i) + "." + "\n" + \
+                self.get_player_total_time(len(history) - i - 2, player) + " >>> " + \
+                self.get_player_total_time(len(history) - i - 1, player) + "\n" + \
+                str(format(history[len(history) - i - 1].time_end - history[len(history) - i - 1].time_start,
```

Comment:
Try reverse indicing i.e. `history[-i - 1]` -> `history[-1] == history[len(history) - 1]`

do we actually need to display time end here?